### PR TITLE
Doc about running a local dashboard through Cloudflare (community)

### PR DESCRIPTION
**Pull request by Ancairon**
##### Summary

A couple of users provided a [very nice set of detailed instructions](https://discord.com/channels/847502280503590932/1154164395799216189/1154556625944854618), and it was decided we will add it to our documentation, in the misc section.

I made some grammar changes, but it should be pretty much as the user suggested it, I just added some bits here and there

### CODE REVIEW

File: docs/running-through-cf-tunnels.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+- Edit `stream.conf` and input:
+
+```ini
+[stream]
+    destination = tcp:127.0.0.1:19999
+```
+
+[Restart the Agents](https://github.com/netdata/netdata/blob/master/docs/configure/start-stop-restart.md), and you are done!
+
+You should now be able to have a Local Dashboard that gets its metrics from Child instnaces, running through Cloudflare tunnels.
```
Comment (hugovalente-pm) [inline docs/running-through-cf-tunnels.md:113]:
```suggestion
You should now be able to have a Local Dashboard that gets its metrics from Child instances, running through Cloudflare tunnels.
```

File: docs/running-through-cf-tunnels.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
+# Running a Local Dashboard through Cloudflare Tunnels
+
+> ### Note
```
Comment (hugovalente-pm) [inline docs/running-through-cf-tunnels.md:3]: This is a nice touch and for sure we should give kudos/reference to where this came from but would probably place this at the end of the page.

**Comment by hugovalente-pm** [comment]
@Ancairon in terms of placing this, where you thinking of it under the streaming section?
image

File: docs/running-through-cf-tunnels.md (hunk 2 above)
Comment (Ancairon) [reply docs/running-through-cf-tunnels.md:3]: that's true, will change

**Comment by Ancairon** [comment]
> @Ancairon in terms of placing this, where you thinking of it under the streaming section?

Along with @MrZammler we were talking putting it under Miscellaneous, because it is too specific a case, do you believe streaming is a better spot?

**Review by MrZammler — APPROVED** [review]
I think under the Misc category is good.

**Review by andrewm4894 — APPROVED** [review]

**Comment by Ancairon** [comment]
I will update the map of Learn and also make a post on reddit, with the same markdown.

**Comment by hugovalente-pm** [comment]
> Along with @MrZammler we were talking putting it under Miscellaneous, because it is too specific a case, do you believe streaming is a better spot?

@Ancairon this is perfect 👌

**Pull request merged**